Mathlib/Order/KrullDimension.lean

```diff
@@ -162,6 +162,37 @@ lemma height_mono : Monotone (α := α) height :=
 @[gcongr] protected lemma _root_.GCongr.height_le_height (a b : α) (hab : a ≤ b) :
     height a ≤ height b := height_mono hab
 
+private lemma height_add_const (a : α) (n : ℕ∞) :
+    height a + n = ⨆ (p : LTSeries α) (_ : p.last = a), p.length + n := by
+  have hne : Nonempty { p : LTSeries α // p.last = a } := ⟨RelSeries.singleton _ a, rfl⟩
+  rw [height_eq_iSup_last_eq, iSup_subtype', iSup_subtype', ENat.iSup_add]
+
+/- For elements of finite height, `height` is strictly monotone. -/
+@[gcongr] lemma height_strictMono {x y : α} (hxy : x < y) (hfin : height y < ⊤) :
```

Comment:
```suggestion
@[gcongr] lemma height_strictMono {x y : α} (hxy : x < y) (hfin : height x < ⊤) :
```
because if `y` has infinite height then its height is greater than `height x`

Reply:
I think I thought multiple times which one is the more general statement, but seems I got it wrong nevertheless :-). Thanks!